source/dom/platform.js

```diff
@@ -45,6 +45,8 @@ enyo.platform = {
 		{platform: "androidChrome", regex: /Android .* Chrome\/(\d+)[.\d]+/},
 		// Android 2 - 4
 		{platform: "android", regex: /Android (\d+)/},
+		// Some android phones, e.g. ZTE and Lenovo
+		{platform: "android", regex: /Android\/(\d+)/},
```

Comment:
if it's common pattern i'd suggest to merge. keep rare/edge case distinct is good for maintaince, i believe.